src/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+function Footer(props) {
```

Comment:
Practice your destructuring here - you could try destructuring the `address` array inside of this component